### ISSUE TITLE
Option to disable modal message "...map styles were updated from..." when a followed template is updated

### ISSUE THREAD

**Issue by hi2u** [freeplane/freeplane#1594]
* I've got a styles template file that I use to define node styles
* And have about 20 other `.mm` files created from it, all of which also have `Follow style changes` enabled
* Every time I open any of my 20 follower `.mm` files after the template was recently changed, I get this modal that I have to click to dismiss:
    * ![2023-12-20_065036 - Freeplane styles updated modal](https://github.com/freeplane/freeplane/assets/9689599/95e581c3-bef6-4f8d-a70a-397500146804)
* Gets very tedious for me seeing I have so many files following it, and I tweak the template pretty often
* Is there any way to disable this modal entirely?
* If not, could an option please be added to let us disable it?
* I think this might be what triggers the modal: 
   * https://github.com/freeplane/freeplane/blob/cd9a36203061c6b6ba8d7cb457c97bd9949c931c/freeplane/src/main/java/org/freeplane/features/styles/MapStyle.java#L689
* Even if the option isn't displayed in the `Preferences...` GUI window, that's fine, don't mind adding a line to the `auto.properties` config file or whatever if needed.

**Comment by acebone** [comment]
I agree - that would be nice. Why not simply put the message in the statusbar?
